### ISSUE TITLE
launch_hbavsslight broken after #177

### ISSUE THREAD

**Issue by smkuls** [initc3/HoneyBadgerMPC#191]
#177 breaks the `launch_hbavsslight.sh` script. The script seems to work fine when this PR is reverted.

**Comment by smkuls** [comment]
@chitianhao

**Comment by nqyy** [comment]
The changes in betterpairing.py in #177 caused this issue. To be specific, in secretshare_hbavsslight.py line 347 to 361:
```
            if await _run_in_thread(self.pc.verify_eval, self.commit,
                                    self.pid+1, self.share, self.witness):
                decryption_time = str(os.times()[4] - decrypt_start_time[4])
                logging.info("decryption time : " + decryption_time)
                self.benchmarkLogger.info(
                    "decryption time :  " + decryption_time)

                self.send_ok_msgs()
                self.sendrecs = True
            else:
                logging.info("verifyeval failed")
                self.send_implicate_msgs()
            while not self.queues["hbavss"].empty():
                (i, o) = self.queues["hbavss"].get_nowait()
                await self.receive_msg(i, o)
````
After the changing, it goes to the "else" which contains an unimplemented method ``send_implicate_msgs()``. Will need @tyurek to take a look.

**Comment by smkuls** [comment]
Another important thing: Why did the tests not catch this? We should first modify the tests to make sure this is caught by them and then propose a fix.

**Comment by amiller** [comment]
changing `asyncio.wait` to `asyncio.gather`  at 
https://github.com/initc3/HoneyBadgerMPC/blob/a0abe5a1ab8094c73fc50919c3797988b5a71d56/tests/test_hbavss.py#L33
makes the test show this error

**Comment by smkuls** [comment]
Closed by #201
